File: 1-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */

var twoSum = function (nums, target) {
  indices = [];
  for (var i = 0; i < nums.length; i++) {
    for (var j = i; j < nums.length; j++)
      if (nums[i] + nums[j + 1] == target) {
        indices.push(i);
        indices.push(j + 1);
        return indices;
      }
  }
};

File: 1_two_sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        indices = []
        for i in range(len(nums)):
            print(i)
            for j in range(i + 1, len(nums)):
                print(j)
                if nums[i] + nums[j] == target:
                    indices.append(i)
                    indices.append(j)
                    return indices